src/Microsoft.Azure.WebJobs.Extension.Storage/Blobs/Listeners/StorageBlobContainerExtensions.cs

```diff
@@ -5,24 +5,23 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Host.Storage.Blob;
```

Comment:
Rename file to CloudBlobContainerExtensions. Rename all the other files as well :)

Reply:
Done